cmd/utils.go

```diff
@@ -186,31 +194,36 @@ func mountExistsLocally(mount string) bool {
 	return fileExists
 }
 
-func getProjectDir() string {
+func getProjectDir() (string, error) {
 	dir, err := os.Getwd()
 	if err != nil {
 		Error.log("Error getting current directory ", err)
-		os.Exit(1)
+		return "", err
 	}
 	appsodyConfig := filepath.Join(dir, ConfigFile)
 	projectDir, err := exists(appsodyConfig)
 	if err != nil {
 		Error.log(err)
-		os.Exit(1)
+		return "", err
 	}
 	if !projectDir {
 
-		Error.log("Current dir is not an appsody project. " +
-			"Run `appsody init <stack>` to setup an appsody project. Run `appsody list` to see the available stacks.")
-
-		os.Exit(1)
+		Debug.log("Current dir is not an appsody project.")
+		// +
+		// "Run `appsody init <stack>` to setup an appsody project. Run `appsody list` to see the available stacks.")
```

Comment:
Reason for not including this text? I'm pretty sure it was added for clarity based on user feedback/issue.